pkg/cli/image/info/info.go

```diff
@@ -139,6 +143,14 @@ func (o *InfoOptions) Run() error {
 			if err != nil {
 				return err
 			}
+			strippedFilter := strings.TrimSuffix(o.FilterOptions.FilterByOS, "$")
```

Comment:
This doesn't seem appropriate.  This should be a method on FilterOptions that gives you singleton platform.

Reply:
right, this was the ugly fix to the `oc image info` i mentioned. I'll improve it

Reply:
updated